### PR TITLE
Embed a precompiled `pprof` binary and launch it in `pprof()`!

### PR DESCRIPTION
**Pull request by NHDaly**
Download the cross-compiled pprof binaries, and launch `pprof` in web-mode when user calls `pprof()`! :)

### CODE REVIEW

**Comment by NHDaly** [comment]
For the record, here is the build-script I used to produce the cross-compiled go pprof binaries:
```julia
cwd = pwd()
GOPATH = ENV["GOPATH"]

# Install pprof
run(`go get -u github.com/google/pprof`)

# Cross compile it
run(`go get -u github.com/mitchellh/gox`)
cd("$GOPATH/src/github.com/google/pprof")
try  # It's okay if this fails for some of the arch × os pairs
    run(`$GOPATH/bin/gox -os="linux freebsd darwin windows" -arch="ppc64 arm arm64 386 amd64" -output="$GOPATH/dist/pprof/pprof_{{.OS}}_{{.Arch}}/pprof"`)
catch end

cd("$GOPATH/dist/pprof")
mkpath("release")
for f in readdir()
    run(`tar -czvf release/$f.tar.gz $f`)
end

println(joinpath(pwd(), "release"))

cd(cwd)
```

File: deps/build.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+if any(!satisfied(p; verbose=verbose) for p in products)
+    try
+        # Download and install binaries
+        url, tarball_hash = choose_download(download_info)
+        install(url, tarball_hash; prefix=prefix, force=true, verbose=true)
+        # NHDALY MANUALLY ADDED THESE LINES TO HOOK UP THE BINARY
+        bin = mkpath(joinpath(prefix, "bin"))
+        dir = splitext(splitext(basename(url))[1])[1]
+        @show dir
+        cp(joinpath(prefix, dir, "pprof"), joinpath(bin, "pprof"))
```
Comment (NHDaly) [inline deps/build.jl]: I added the above 4 lines manually to get the `pprof` binary out of the extracted `.tar.gz` and into the `bin` dir. This probably needs to be fixed

**Comment by NHDaly** [comment]
Bummer, it looks like the tests fail, complaining:
```
Failed to execute dot. Is Graphviz installed?
exec: "dot": executable file not found in $PATH
```
So we may need to install more stuff than just pprof... it might automatically install other deps?

**Comment by vchuravy** [comment]
:)

**Comment by NHDaly** [comment]
K well, i've added a sentence to the README that if you want to use the web view, you need graphviz. It seems to me that almost everyone has it, so this shouldn't be a big deal! :)

Merging this now!

**Pull request merged**